extensions/blocks/instagram-gallery/edit.js

```diff
@@ -58,22 +60,37 @@ const InstagramGalleryEdit = props => {
 		apiFetch( {
 			path: addQueryArgs( '/wpcom/v2/instagram/gallery', {
 				access_token: accessToken,
-				count,
+				count: 30,
```

Comment:
I wonder if we should move this to a const too, since we use it more than once...